### DISCUSSION TITLE
skoala/quickstart/install/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1496] [Announcements]
# skoala/quickstart/install/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/quickstart/install/

**Comment by liyuerich** [comment]
商业版 ≤ v0.3.28

我们的商业版已经0.7.0了，是不是较早部分的特别处理可以归档？不在最新文档中显示？

**Reply by samzong** [reply]
文档表述无误，0.3.28 之前的确不会自动安装，需要手工特殊处理。

**Reply by Michelle951** [reply]
improved by pull request: https://github.com/DaoCloud/DaoCloud-docs/pull/1885

**Comment by liyuerich** [comment]
helm repo add skoala-release https://release.daocloud.io/chartrepo/skoala

这个命令是在线模式，能都给个离线模式的命令？

**Reply by samzong** [reply]
已在离线文档中有描述，请移步查看离线文档 👉 <https://docs.daocloud.io/skoala/quickstart/offline-upgrade/>

**Comment by liyuerich** [comment]
微服务引擎部署结构， skoala-init 结构图片中有个 skoala=agent ？ 应该是skoala-agent吧？

**Reply by samzong** [reply]
`skoala-init` 包含了 `skoala` 在工作集群需要的组件，组件内包含了 `skoala-agent`， 配图无误。

**Reply by liyuerich** [reply]
我是说skoala 和agent中间应该是 - ， 图片上用了 “等号=”

**Reply by Michelle951** [reply]
fixed by pull request: https://github.com/DaoCloud/DaoCloud-docs/pull/1885

**Comment by kebe7jun** [comment]
```
~ helm upgrade --install skoala --create-namespace -n skoala-system --cleanup-on-fail \
    --set ui.image.tag=v0.9.0 \
    --set sweet.enable=true \
    --set hive.configMap.data.database.host=mcamel-common-mysql-cluster-mysql-master.mcamel-system.svc.cluster.local \
    --set hive.configMap.data.database.port=3306 \
    --set hive.configMap.data.database.user=root \
    --set hive.configMap.data.database.password=xxxxxxxx \
    --set hive.configMap.data.database.database=skoala \
    skoala-release/skoala \
    --version 0.13.0

```

seems out of date.

**Comment by samzong** [comment]
DME 是什么？
